src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Internal/ModelBindingHelper.cs

```diff
@@ -120,41 +85,6 @@ public static class ModelBindingHelper
             params Expression<Func<TModel, object>>[] includeExpressions)
            where TModel : class
         {
-            if (model == null)
```

Comment:
It's just noise when the parameters are checked in the longest overload. We aren't consistent about removing duplication but definitely do it.